apps/desktop-wallet/src/components/DataList.tsx

```diff
@@ -19,23 +19,19 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 import { ReactNode } from 'react'
 import styled from 'styled-components'
 
-import { validateChildrenType } from '@/utils/misc'
-
 interface DataListProps {
   children: ReactNode
   title?: string
 }
 
-const DataList = ({ children, title }: DataListProps) => {
-  validateChildrenType({ children, childType: DataListRow, parentName: 'DataList' })
+const DataList = ({ children, title }: DataListProps) => (
+  // validateChildrenType({ children, childType: DataListRow, parentName: 'DataList' })
```

Comment:
I could not find a way to create a component in a separate file that is a `DataList.Row` without having this validation screening in the console. Any help is appreciated.